### DISCUSSION TITLE
part-ii/2008-66/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#240] [Question comments]
Part II 2008 Graph Theory: 3.II $. 17 \mathrm{~F} \quad$ 

https://questions.tripos.org/part-ii/2008-66/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The first part is bookwork.

For the tree, we can prove the formula by induction on $n$; since a tree always has a leaf vertex, we can remove that leaf vertex, assume the formula for $n-1$, and add the leaf vertex back (and there are $t-1$ possible colours for it since it is only adjacent to one other vertex). 

The converse is true. With regards to the hint, the chromatic polynomial of $G$ is equal to the product of the chromatic polynomials of its components. Now the chromatic polynomial of a graph always has a factor of $t$, i.e. a constant term of $0$ (this can be shown by induction on $e(G)$ with the deletion-contraction relation, and an interpretation of this is that no graph can be coloured with $0$ colours), so if a graph has $k$ connected components, it has a factor of $t^k$. 

Now using this result and the earlier results from the question, if $G$ has chromatic polynomial $t(t-1)^{n-1}$, then $G$ is a connected graph with $n$ vertices and $n-1$ edges. But this forces $G$ to be a tree.

For the Turan graph, if a graph $G$ has the same chromatic polynomial as $T_{r}(n)$, then we know that $G$ has $n$ vertices and $e(T_{r}(n))$ edges, and contains no $K_{r+1}$ (since $G$ is $r$-colourable, which can be determined by evaluating the chromatic polynomial at $t=r$ and showing that we get a positive integer result). But in our proof of Turan's theorem, we showed that any graph with these properties is isomorphic to $T_{r}(n)$, so we are done.

</details>
